code/game/objects/items.dm

```diff
@@ -143,6 +143,8 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	/// Is this item fryable without a syndicate frying pan
 	var/fryable = FALSE
 
+	var/printed = FALSE
```

Comment:
Maybe just add a trait for that?

Reply:
> Maybe just add a trait for that?

You mean an ITEM/OBJ Flag?

Reply:
That one that is added through ADD_TRAIT

Reply:
do that

Reply:
yeah, probably best that not every item has an additional variable to track